### ISSUE TITLE
training from scratch, loss decreased very slow

### ISSUE THREAD

**Issue by chrisx599** [facebookresearch/TimeSformer#137]
have anyone who training from scratch, not use pre-trained weight of ViT-B?
Can author or anyone released the training log of train from scratch？
the author said it takes more epochs to training from scratch, but did'nt released a concrete number, can author says this number?
**when i trained 15 epochs, the accuracy is only 16%, and from the beginning, it's just 4%, i'm so confused is that normal?**
![image](https://github.com/user-attachments/assets/fa165219-3d37-4477-ad70-89f854effdc9)
![image](https://github.com/user-attachments/assets/a34aa7e9-feed-4aa3-983d-ebb3f53cf53c)

**Comment by chrisx599** [comment]
first figure is loss, seconde figure is accuracy
